src/Powershell/NewEnvironment.cs

```diff
@@ -17,8 +17,7 @@ public class NewEnvironment : ReplaceTokensBaseCmdlet<NewEnvironmentHandler>
         public string Name { get; set; }
 
         [Parameter(
-            HelpMessage = Resources.DEPLOYMENT_PARAMETERSPATH_HELP,
-            Mandatory = true
+            HelpMessage = Resources.DEPLOYMENT_PARAMETERSPATH_HELP

```

Comment:
Is there a reason for this to start being optional?

Reply:
Answer to myself: **parameters** are no longer mandatory, they may be set using the **interactive** session